storage/innobase/include/fil0crypt.h

```diff
@@ -612,6 +612,52 @@ fil_space_get_scrub_status(
 	const fil_space_t*		space,
 	fil_space_scrub_status_t*	status);
 
+/** Handles the fetching/generation/storing/etc of keyring redo log keys.
+ 
+ This class is *NOT* thread safe, as thread safety is not required.
+ Data is only accessed/modified on the following points:
+ * When the redo space is created, at startup
+ * During redo log recovery, at startup
+ * When the server UUID is generated, at startup
+ * When the user requests a new key version, checked periodically in the
+   master thread
+ 
+ As these can't happen in parallel, no lock is used.
+ */
+class redo_log_keys {
+       public:
+	MY_NODISCARD
```

Comment:
Does it have be seriously on three lines? One line is better IMHO

Applicable to all lines below